### PR TITLE
Added WebsiteArticleController; Modified documentation; Added example files

### PR DESCRIPTION
**Pull request by trickreich**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Added default WebsiteArticleController.
Better README for configuring bundle.
Added examples for structure and template.

### CODE REVIEW

File: Controller/WebsiteArticleController.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    public function indexAction($view, ArticleDocument $object, Request $request)
+    {
+        if ($object->getAuthored() > new \DateTime()) {
+            throw $this->createNotFoundException();
+        }
+
+        $categoryManager = $this->get('sulu_category.category_manager');
+
+        $content = $this->resolve($object);
+        $categories = $categoryManager->findByIds($content['extension']['excerpt']['categories']);
```
Comment (wachterjohannes) [inline Controller/WebsiteArticleController.php]: this is not needed in all cases so remove it-

File: Controller/WebsiteArticleController.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+        return $this->render(
+            $view . '.html.twig',
+            array_merge(
+                [
+                    'categories' => $categoryManager->getApiObjects($categories, $request->getLocale())
```
Comment (wachterjohannes) [inline Controller/WebsiteArticleController.php]: remove

File: Controller/WebsiteArticleController.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+     */
+    public function indexAction($view, ArticleDocument $object, Request $request)
+    {
+        if ($object->getAuthored() > new \DateTime()) {
+            throw $this->createNotFoundException();
+        }
+
+        $categoryManager = $this->get('sulu_category.category_manager');
+
+        $content = $this->resolve($object);
```
Comment (wachterjohannes) [inline Controller/WebsiteArticleController.php]: call serialized directly.

File: Controller/WebsiteArticleController.php [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    /**
+     * @param string $view
+     * @param ArticleDocument $object
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function indexAction($view, ArticleDocument $object, Request $request)
+    {
+        if ($object->getAuthored() > new \DateTime()) {
```
Comment (wachterjohannes) [inline Controller/WebsiteArticleController.php]: this shouldn't be here.

**Comment by wachterjohannes** [comment]
@trickreich also styleci is failing

**Comment by trickreich** [comment]
@wachterjohannes Can you have a look once more?

**Pull request merged**